### ISSUE TITLE
Small device aside navigation is not working.

### ISSUE THREAD

**Issue by itsmnthn** [nuxt-themes/docus#699]
[Repository](https://github.com/hubble-exchange/docs)
[Live preview](https://v2.docs.hubble.exchange/)

![image](https://user-images.githubusercontent.com/25129674/204864794-b2b74fe1-2417-495a-984f-7a321b706d5a.png)

Using `app.config.ts`

```
export default defineAppConfig({
  docus: {
     header: { logo: true, showLinkIcon: false, exclude: ["/docs", "/blogs"] },
     aside: { level: 1, collapsed: false, exclude: [] },
  }
})
```

output of `npx nuxi info`:

```
- Operating System: `Darwin`
- Node Version:     `v18.12.1`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `extends`, `components`, `content`
- Runtime Modules:  `-`
- Build Modules:    `-`
```

**Comment by Tahul** [comment]
@bdrtsky : could you look into this?

**Comment by itsmnthn** [comment]
@bdrtsky I've updated the config to not exclude the form header in the latest commits. Check this config [app.config.ts](https://github.com/hubble-exchange/docs/blob/10e95ac27693cff8657caf226c5bd0e45bb740b5/app.config.ts) and [live preview](https://638857b03a535300093c81bf--docs-hubble-exchange.netlify.app/) at this commit.

To fix this issue I've created pr with fix #710

**Comment by Tahul** [comment]
Hey @itsmnthn ; will look into your PR very soon!
